resources/design-document.md

```diff
@@ -4,188 +4,185 @@
+* Creating transactions representing incoming and outgoing item shipments, and adjusting on-hand inventory item numbers accordingly
+* Generating HTML inventory shipping reports with custom time periods
+* Using third-party API to generate and download PDF document of inventory report
 
-### 4.1. In Scope
+### 3.2. Out of Scope
 
-* Creating inventory items, updating their metrics, and removing them when not in any warehouse
-* Creating, populating, and arranging new warehouses
-* Updating inventory, generating reports, adding and removing items from existing warehouses, and rearranging existing warehouses
+* Individual item reports
+* Configurable warehouse sizes and layouts (the initial product will use the same basic warehouse size/floor plan for all warehouses)
 
-### 4.2. Out of Scope
+### 3.3 Stretch Goals
+* Automatic warehouse layout creation and display, using machine learning to generate efficient layouts
+* Item recommendation service, which can autopopulate a warehouse based on data metrics
+* Customer information table, with endpoints and front-end pages allowing the creation of customers and pulling of customer shipping reports.
+* Supplier information table, with endpoints and front-end pages allowing the creation of suppliers and pulling of supplier stocking reports
 
-* Automatic updating of inventory item metrics based on stock changes
-* Individual item reports
-* Configurable warehouse spaces and layouts (the initial product will use the same basic floor plan for all warehouses)
+# 4. Proposed Architecture Overview
 
-# 5. Proposed Architecture Overview
+The initial product will provide the minimum lovable product (MLP) including creating, updating, and deleting warehouses. 
+It will also allow adding and removing items to warehouse inventories, tracking their quantities on hand, tracking incoming and outgoing shipments, and generating time-based shipping reports.
 
-The initial product will provide the minimum lovable product (MLP) including creating, updating, and deleting inventory items, updating their metrics, spinning up new warehouses, selecting their products and arranging the items on the available shelf-space.
+This will be accomplished using a RESTful API Gateway and AWS Lambda to  create endpoints.  The product stack will be deployed into an S3 bucket using CloudFormation
 
-This will be accomplished using a RESTful API and AWS Lambda to create endpoints.
+inventoryItem, transaction, and warehouse data will be stored in DynamoDB
 
-Inventory item and warehouse data will be stored in DynamoDB
+PopStock will also provide a web interface for users to manage their warehouses, and inventories.  There will be a main page showing existing warehouses, with buttons to create a new one or to modify and existing one.
+Clicking the button for an existing warehouse will lead to a page where you can update a warehouse's inventory, with buttons leading to pages for Inventory Item creation and shipping reports
```

Comment:
Not a blocker to design approval, but might want to check the problem statement and use cases to see if anything should be update with the revised MVP scope.